### PR TITLE
Market Preview: Categorical Outcomes

### PR DESCRIPTION
**Pull request by sharkcrayon**
Created the layout for categorical outcomes on the market preview card.

### CODE REVIEW

File: src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/* eslint react/no-array-index-key: 0 */  // It's OK in this specific instance as order remains the same
```
Comment (stephensprinkle) [inline src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx]: Safe to remove this linting override

File: src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import getValue from 'utils/get-value'
+
+import Styles from 'modules/market/components/market-outcomes-categorical/market-outcomes-categorical.styles'
+
+const CategoricalOutcome = ({ className, outcome }) => (
+  <div className={className || Styles.MarketOutcomesCategorical__outcome}>
+    <span className={Styles['MarketOutcomesCategorical__outcome-name']}>{outcome.name}</span>
+    <span className={Styles['MarketOutcomesCategorical__outcome-value']}>{getValue(outcome, 'lastPricePercent.full')}</span>
+  </div>
+  )
```
Comment (stephensprinkle) [inline src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx]: Indentation

**Comment by sharkcrayon** [comment]
@stephensprinkle Fixed and ready to go!

File: src/modules/market/components/market-basics/market-basics.jsx [hunk 3]
```diff
@@ -58,7 +59,11 @@ const MarketBasics = (p) => {
         </h1>
 
         {(p.type === BINARY || p.type === SCALAR) &&
-          <MarketPreviewOutcomes outcomes={p.outcomes} min={p.minValue} max={p.maxValue} type={p.type} />
+          <MarketOutcomesBinaryScalar outcomes={p.outcomes} min={p.minValue} max={p.maxValue} type={p.type} />
```
Comment (stephensprinkle) [inline src/modules/market/components/market-basics/market-basics.jsx]: Generally you’ll want to line break attributes for readability if there are more than one.

File: src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    return (
+      <div className={Styles.MarketOutcomesCategorical} style={outcomeWrapperStyle}>
+        { outcomes.length > 0 &&
+          <CategoricalOutcome className={Styles['MarketOutcomesCategorical__height-sentinel']} outcome={outcomes[0]} />
+        }
+        <div className={classNames(Styles['MarketOutcomesCategorical__outcomes-container'], { [`${Styles['show-more']}`]: displayShowMore })}>
+          { displayShowMore &&
+            <button className={Styles['MarketOutcomesCategorical__show-more']} onClick={this.showMore}>{ showMoreText }</button>
+          }
+          <div className={Styles.MarketOutcomesCategorical__outcomes} ref={(outcomeTable) => { this.outcomeTable = outcomeTable }}>
```
Comment (stephensprinkle) [inline src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx]: When line breaking attributes, place the `ref` first.

File: src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+MarketOutcomesCategorical.propTypes = {
+  outcomes: PropTypes.array.isRequired,
+  max: PropTypes.string,
```
Comment (stephensprinkle) [inline src/modules/market/components/market-outcomes-categorical/market-outcomes-categorical.jsx]: Seems like you don’t use these at all in the component

**Pull request merged**